toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -468,11 +468,21 @@ func (r *RpmRepoCloner) WhatProvides(pkgVer *pkgjson.PackageVer) (packageNames [
 				return
 			}
 
+			// Flag the result if we did case-insensitive lookup.
+			lookupIgnoredCase := tdnf.DidCaseInsensitiveMatchRegex.MatchString(stdout)
+
 			// MUST keep order of packages printed by TDNF.
 			// TDNF will print the packages starting from the highest version, which allows us to work around an RPM bug:
 			// https://github.com/rpm-software-management/rpm/issues/2359
 			for _, matches := range tdnf.PackageLookupNameMatchRegex.FindAllStringSubmatch(stdout, -1) {
 				packageName := matches[tdnf.PackageNameIndex]
+				if lookupIgnoredCase {
+					logger.Log.Warnf("'%s' was found by case-insensitive lookup of '%s', but this is not a valid (Build)Requires and will be ignored", packageName, pkgVer.Name)
```

Comment:
I would not mention (Build)Requires in a log message. This loosely ties this tool to RPM specs' terminology, while the tool itself can be used in more generic cases to find package providing a capability for other scenarios. Suggestion:
```suggestion
					logger.Log.Warnf("'%s' was found by case-insensitive lookup of '%s', but this is not valid and will be ignored", packageName, pkgVer.Name)
```